backend/tools/keycloak/sync/appsettings.json

```diff
@@ -15,6 +15,7 @@
     }
   },
   "Sync": {
+    "Claims": [ "system-administrator", "agency-administrator", "admin-agencies", "admin-users", "admin-roles", "property-view" ],
```

Comment:
Need to setup initial roles/claims or an entirely new environment won't work at all.